oidc-controller/api/routers/acapy_handler.py

```diff
@@ -54,9 +54,24 @@ async def post_topic(request: Request, topic: str, db: Database = Depends(get_db
                     if sid:
                         await sio.emit("status", {"status": "verified"}, to=sid)
                 else:
-                    auth_session.proof_status = AuthSessionState.FAILED
+                    client = AcapyClient()
+                    logger.info(
+                        f"MY RESULT {webhook_body['by_format']['pres']['indy']['identifiers'][0]['rev_reg_id']}"
+                    )
+                    rev = client.is_revoked(
+                        webhook_body["by_format"]["pres"]["indy"]["identifiers"][0][
+                            "rev_reg_id"
+                        ]
+                    )
+                    logger.info(f"Credential was revoked {rev}")
```

Comment:
This log statement might be best as a `warning`, so that it is highlighted in the logs if someone goes looking for information. Also wondering if there can be more details associated to the actual message, in order to determine which presentation exchange/credential triggered it.